dnadoh/recombine.py

```diff
@@ -0,0 +1,56 @@
+"""Recombination of the genomes of two people
+to produce a third person. Other features are
+averaged with some noise.
+"""
+
+from typing import List
+from .synthesize import PersonGenerator, Person
+
+def recombine(
+    person1: Person,
+    person2: Person,
+    reference_genome: str,
+    pid: str,
+    recombination_prob: List[float],
+    recombination_loci: List[int]
```

Comment:
just making sure I understand this correctly - is this a list of the positions within the genome?

(Note to self, we should start adding doc-strings -- not necessary for you now though since I know every other function in our code doesn't have any 😂 )

Reply:
Added docstring! 😄